proto/backend.proto

```diff
@@ -87,162 +107,50 @@ message DataQueryResponse {
   map<string,string> metadata = 2;
 }
 
-//-----------------------------------------------
-// HTTP style request
-//-----------------------------------------------
-
-message CallResource {
-  message StringList {
-    repeated string values = 1;
- }
-
-  message Request {
-    PluginConfig config = 1;
-    string path = 2;
-    string method = 3;
-    string url = 4;
-    map<string,StringList> headers = 5;
-    bytes body = 6;
-    User user = 7;
-  }
-
-  message Response {
-    int32 code = 1;
-    map<string,StringList> headers = 2;
-    bytes body = 3;
-  }
-}
-
 //-----------------------------------------------
 // Diagnostics
 //-----------------------------------------------
 
 service Diagnostics {
```

Comment:
Wasn't this suppose to be renamed `Core`? Since all plugins are expected to be able to expose metrics and be "healthcheckable"

Reply:
Not that I remembered since we later agree upon splitting into Data and Resource.